utils/message_html_to_component.jsx

```diff
@@ -130,6 +132,8 @@ export function messageHtmlToComponent(html, isRHS, options = {}) {
                     <MarkdownImage
                         className={className}
                         imageMetadata={options.imagesMetadata && options.imagesMetadata[attribs.src]}
+                        height={parseInt(height, 10)}
```

Comment:
Funny that you hit this right when I did in https://github.com/mattermost/mattermost-webapp/pull/8622. These actually need to be strings since they can be either the empty string or "auto". Do my changes fix the issue you were running into with passing them as strings?

Reply:
I am pretty sure your solution deals with the string problem (those were mainly about annoying warnings in the webapp). I will revert the related changes and focus only on the toggle problem.